app/lib/supergroups_api/supergroup_collection.rb

```diff
@@ -0,0 +1,29 @@
+module SupergroupsApi
+  class SupergroupCollection
```

Comment:
There are a lot of classes with "Supergroup" in the name. What is a "SupergroupCollection"?

Reply:
SupergroupCollection is a collection of Supergroup instances. Do you think it will be more understandable as `Search::Supergroups`?

Reply:
And a Supergroup is the result of a search by supergroup and organisation?

Reply:
There's already a Supergroup module that's used to power the Taxon pages: https://github.com/alphagov/collections/blob/master/app/presenters/supergroups/guidance_and_regulation.rb

Reply:
The different Rummager searches seem to live here: https://github.com/alphagov/collections/tree/master/app/services, so you might want to consider following the same pattern.

Reply:
Yeah that is a good idea.